src/ray/gcs/redis_accessor.cc

```diff
@@ -25,6 +25,26 @@ namespace gcs {
 RedisActorInfoAccessor::RedisActorInfoAccessor(RedisGcsClient *client_impl)
     : client_impl_(client_impl), actor_sub_executor_(client_impl_->actor_table()) {}
 
+std::vector<ActorID> RedisActorInfoAccessor::GetAllActorID() const {
+  return client_impl_->actor_table().GetAllActorID();
+}
+
+Status RedisActorInfoAccessor::GetActorData(const ActorID &actor_id,
```

Comment:
rename to `Get`, since the async interface is `AsyncGet`.

Reply:
> rename to `Get`, since the async interface is `AsyncGet`.

Okey, make sense.